conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/tezos/TezosTupleFlattenHelper.scala

```diff
@@ -1,10 +1,10 @@
-package tech.cryptonomic.conseil.api.routes.openapi
+package tech.cryptonomic.conseil.api.routes.platform.data.tezos
 
 /** Helper object which implements flatten for the tuples using shapeless library
   * Taken from:
   * https://gist.github.com/travisbrown/3945529
   * */
-object TupleFlattenHelper {
+private[tezos] object TezosTupleFlattenHelper {
```

Comment:
This shouldn't be tezos-specific, am I missing something?
I would expect this to be a common helper

Reply:
You are actually right.

Reply:
I've moved that to `util` package in `common`